_projects/it/siti-comuni.md

```diff
@@ -0,0 +1,62 @@
+#payoff: Il punto di riferimento per il design dei siti web dei Comuni
+logo: 
+lang: it
+order: 4
+highlight: true
+layout: project
+---
+
+### Descrizione
+Designers Italia, la community di design dei servizi pubblici ideata dal [Team Digitale](https://teamdigitale.governo.it/), sta realizzando, in collaborazione con [AGID](https://www.agid.gov.it/) un modello per la realizzazione di siti web per gli oltre 8mila Comuni italiani, basato sul proprio *design system*. 
+
+### Obiettivi
+
+Il modello propone uno standard a tutti i Comuni che vogliono rendere più semplice ed efficace il proprio sito, sulla base degli effettivi bisogni dei cittadini. L’idea è che il lavoro di progettazione di ciascun Comune possa partire da componenti standard testate e verificate con gli utenti, in modo che ciascun Comune possa concentrarsi sugli elementi di unicità del servizio/della città, senza dover reinventare ogni volta la ruota.
+
+### L’attività di ricerca
+
+Il modello si basa su un’attività di analisi desk partita nel 2017, su una ricerca “sul campo” condotta da un team dedicato (interviste e test di usabilità realizzati nei mesi di giugno e luglio 2018) e sulla collaborazione di alcuni Comuni italiani. In particolare il **Comune di Firenze** ha messo a disposizione il materiale e l’esperienza alla base del nuovo sito comunale (a sua volta basato sul *design system* di Designers Italia); la Regione Veneto, nell’ambito del progetto MyPortal, ha messo a disposizione i risultati di un’attività di ricerca con i cittadini dei Comuni della Regione (utilizzando il protocollo di Designers Italia).
+
+#### Il metodo di lavoro
+
+[Il kit di Designers Italia per i test di usabilità](https://designers.italia.it/kit/web-toolkit/)
+[Il kit per le user interview](https://designers.italia.it/kit/user-interviews/)
+[Guarda tutti i kit di Designers Italia](https://designers.italia.it/kit/)
+
+#### I risultati della ricerca
+
+I risultati delle attività di ricerca verranno pubblicati nel corso del mese di settembre. 
+
+Qui alcune attività preliminari di analisi condotte nel 2017 e nel 2018:
+
+[Data-driven design: ecco come i dati possono aiutarci a capire i bisogni dei cittadini](https://medium.com/designers-italia/data-driven-design-ecco-come-i-dati-possono-aiutarci-a-capire-i-bisogni-dei-cittadini-38f04d898f4d)
+[Cosa fanno gli utenti del sito di un Comune? Ce lo dice Piwik](https://medium.com/designers-italia/cosa-fanno-gli-utenti-del-sito-di-un-comune-ce-lo-dice-piwik-dedc67504b35)
+[Card sorting e navigazione per i siti web dei Comuni](https://medium.com/designers-italia/card-sorting-e-navigazione-per-i-siti-web-dei-comuni-f4ae175b84c4)
+
+### Il prototipo
+
+#### Il metodo di lavoro
+
+[Linee guida di design per i servizi digitali della Pubblica Amministrazione](https://designers.italia.it/guide/)
+[Il wireframe kit](https://designers.italia.it/kit/ui-kit/)
```

Comment:
https://designers.italia.it/kit/wireframe-kit/ 😉

Reply:
!!! :) a te